packages/@glimmer/manager/lib/internal/api.ts

```diff
@@ -5,6 +5,7 @@ import type {
   Owner,
 } from '@glimmer/interfaces';
 import { debugToString } from '@glimmer/debug-util';
+import { debugAssert } from '@glimmer/global-context';
```

Comment:
Lots of little clarifying tweaks in this file, including using asserts instead of errors.